.prettierignore

```diff
@@ -0,0 +1,2 @@
+**/*.js
```

Comment:
Can you please remove this file, I don't want to clutter the root directory too much.

Reply:
Yeah sorry that was a mistake, just had to stop vscode from reformatting everything & forgot to omit it from commit